CurlAndLiquid.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:18.04
+
+ARG RECV_KEYS=''
+
+ENV LANG='en_US.UTF-8' \
+    LANGUAGE='en_US:en'
+
+# Install Curl
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+            curl \
+            ca-certificates \
+            unzip \
+            gnupg \
+            dirmngr \
+    \
+    && rm -r /var/lib/apt/lists/*
+
+# Install Ruby and RVM
+RUN gpg --keyserver hkp://keys.gnupg.net:80 --keyserver-options http-proxy=${http_proxy} --recv-keys=${RECV_KEYS} && \
```

Comment:
since we already use the style in this repo, please keep ampersands on the new line, instead of at the end